.circleci/config.yml

```diff
@@ -55,7 +55,7 @@ jobs:
                       echo "//registry.yarnpkg.com/:_authToken=$NPM_TOKEN" >> ~/.npmrc
             - run:
                   name: Publish @brightlayer-ui/icons
-                  command: cd packages/icon-font && yarn publish:package -b $CIRCLE_BRANCH
```

Comment:
This cd step is required (there is no publish:package command in the root folder, nor should there be). You need to step into each package subfolder to execute the publish command.

Reply:
Notice the error:
<img width="1138" alt="image" src="https://user-images.githubusercontent.com/29152776/209129446-49973e44-ef43-4d39-bc88-e410b0dfad9c.png">

It's an issue with the node_modules. You probably need to add the `.yarn` folder to the list of folders that persist from the build job.